api/tests/unit/domain/validators/schooling-registration-validator_test.js

```diff
@@ -142,41 +166,73 @@ describe('Unit | Domain | Schooling Registration validator', () => {
       });
 
       it('is valid when birthCity is undefined and birthCountry is France', async () => {
-        const franceCountryCode = '99100';
-
         try {
-          checkValidation({ ...validAttributes, birthCountryCode: franceCountryCode, birthCity: undefined });
+          checkValidation({ ...validAttributes, birthCountryCode: FRANCE_COUNTRY_CODE, birthCityCode: '51430', birthCity: undefined });
         } catch (e) {
           expect.fail('SchoolingRegistration is valid when birthCity is undefined and birthCountry is France');
         }
       });
     });
 
     context('birthCityCode', () => {
-      it('throw an error when birthCityCode is not France and birthCity is undefined', async () => {
-        const franceCountryCode = '99100';
-
-        const error = await catchErr(checkValidation)({ ...validAttributes, birthCountryCode: franceCountryCode, birthCityCode: undefined });
-
-        expect(error.key).to.equal('birthCityCode');
-        expect(error.why).to.equal('required');
-      });
-
-      it('is valid when birthCityCode is undefined and birthCountry is not France', async () => {
+      context('is valid', () => {
+        context('when birthCountryCode equal to France', () => {
+
+          it('respects INSEE Code, with one letter', async () => {
+            try {
+              checkValidation({ ...validAttributes, birthCountryCode: FRANCE_COUNTRY_CODE, birthCityCode: '2B125' });
+            } catch (e) {
+              expect.fail('SchoolingRegistration is valid birthCityCode respect INSEE code, like Corsica');
+            }  
+          });
+    
+          it('respects INSEE Code, only number', async () => {
+            try {
+              checkValidation({ ...validAttributes, birthCountryCode: FRANCE_COUNTRY_CODE, birthCityCode: '2B125' });
```

Comment:
je l'avais pas vue celle-là ^^ pour coller avec le titre du test, c'est mieux (et l'autre est déjà testé plus haut) :
```suggestion
              checkValidation({ ...validAttributes, birthCountryCode: FRANCE_COUNTRY_CODE, birthCityCode: '21125' });
```